apis/v1alpha1/local_object_reference_types.go

```diff
@@ -92,9 +101,10 @@ type SecretsDefaultLocalObjectReference struct {
 	//   resource: foos
 	//   name: myfoo
 	//
-	// +optional
+	// +kubebuilder:validation:MinLength=1
```

Comment:
min can be removed sine the field is being defaulted.

Reply:
Updated, thanks!